MindblownHQ/shop-health
# Code Styling and Static Analysis

# Summary
This PR adds [Yoast CS](https://github.com/Yoast/yoastcs) and [PHPStan](https://phpstan.org/) to the repository.

# Technical details
Yoast CS is a Code Style check based on PHPCS and WPCS. It will help and force us to keep a cleaner and modern code base.

PHPStan is a static analysis tool that will help us find bugs without actually writing tests. It is the first step in preventing releasing a product that might have, for instance, missing variables, functions and such.

# Test instructions
- Run `composer install`, this will install the YoastCS and PHPStan packages
- For CS On the command line run: `composer phpcs`
- For PHPStan run: `composer analyse`

Both should start a script that either returns a list of errors of an all okay message.
